.docker/base/add-repo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+#Prepare everest-ci and everest-logs repositories
+cd $MYHOME
+git clone git@github.com:/project-everest/everest-ci
```

Comment:
does github allow a leading `/`? I would've assumed that the URL had to be of the form `git@github.com:project-everest/...`